### PR TITLE
fix(axios-ext): encode transport request payload

### PR DESCRIPTION
**Pull request by cianmSAP**
#1083

Encodes the path in case there is a namespace

### CODE REVIEW

**Review by longieirl — APPROVED** [review]
Looks good to me.
- tests cover the change.

**Review by longieirl — APPROVED** [review]
Code smell addressed.

File: .changeset/chilled-geese-sniff.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/axios-extension': minor
```
Comment (tobiasqueck) [inline .changeset/chilled-geese-sniff.md:2]: patch should be fine since your fixing a bug

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
- change is clear and easy to understand
- tests cover the fix
- please change the changeset from minor to patch since it is a bug fix and not a new feature

File: .changeset/chilled-geese-sniff.md (hunk 1 above)
Comment (cianmSAP) [reply .changeset/chilled-geese-sniff.md]: Updated changeset, thanks @tobiasqueck

**Review by tobiasqueck — APPROVED** [review]

**Review by longieirl — APPROVED** [review]

**Pull request merged**